app/services/broadcast_collection_change.rb

```diff
@@ -0,0 +1,15 @@
+# typed: strict
+# frozen_string_literal: true
+
+# Sends messages about a collection changing state to the CollectionUpdatesChannel
+class BroadcastCollectionChange
+  extend T::Sig
+
+  sig { params(collection: Collection, state: Symbol).void }
+  def self.call(collection:, state:)
+    return unless state == :deposited
```

Comment:
Can you say more about the differences between this service and the one for works? E.g., the guard clause on line 10, and broadcasting the state as `''`?

Reply:
Collections only show "depositing" otherwise they don't show a stare

Reply:
I've added a comment to this effect.